frontend/packages/policy-editor/src/components/PolicyEditor.tsx

```diff
@@ -122,6 +126,7 @@ export const PolicyEditor = ({
               : false
```

Comment:
The showErrorCode above can be simplified like this. :) 

`showErrors={
  showAllErrors ||
  (showErrorsOnAllRulesAboveNew && policyRules.length - 1 !== i)
}`